### ISSUE TITLE
Not able to run dropEst via Centos7 Docker

### ISSUE THREAD

**Issue by TomSmithCGAT** [hms-dbmi/dropEst#64]
Hi @VPetukhov. I had some issues compiling `dropEst` so I've built and started the docker container as indicated in the installation instructions:

```
cd dropEst/dockers/centos7
docker build -t dropest .
docker run --name dropest -it dropest
```

Then downloaded a test BAM from 10X - `https://support.10xgenomics.com/single-cell-gene-expression/datasets/2.1.0/neurons_900`

When I run dropEst using `dropest -f -c dropEst/configs/10x.xml ./neurons_900_possorted_genome_bam.bam -o test`, I get the following error:

```
Compiling diagnostic stats: 
Reads per chromosome per cell;
Fill exon results
Error: function 'dataptr' not provided by package 'Rcpp'
In addition: Warning message:
In library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
  there is no package called 'Rcpp'
Execution halted
```

I checked that `Rcpp` is definitely in the R library (`R/x86_64-redhat-linux-gnu-library/3.4/`) and can be attached. I note that the R version (`3.5.1`) doesn't align with the version number indicated in the docker R library (`3.4`) but no idea if that's an issue. I'm not up to scratch with `Rcpp` so I'm not sure what the above error means.

[EDIT]: attached `est_main.log` and `est_config.dump.xml` in case that helps...
[est_main.log](https://github.com/hms-dbmi/dropEst/files/2737711/est_main.log)
[est_config.dump.xml.txt](https://github.com/hms-dbmi/dropEst/files/2737715/est_config.dump.xml.txt)

```
$R
R version 3.5.1 (2018-07-02) -- "Feather Spray"
Copyright (C) 2018 The R Foundation for Statistical Computing
Platform: x86_64-redhat-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.
> library("Rcpp")
> sessionInfo()
R version 3.5.1 (2018-07-02)
Platform: x86_64-redhat-linux-gnu (64-bit)
Running under: CentOS Linux 7 (Core)

Matrix products: default
BLAS/LAPACK: /usr/lib64/R/lib/libRblas.so

locale:
[1] C

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] Rcpp_1.0.0

loaded via a namespace (and not attached):
[1] compiler_3.5.1
```

**Comment by ravipatel4** [comment]
@TomSmithCGAT, I am facing a same problem. Were you able to find a solution? Highly appreciate any help. Thanks in advance.

**Comment by VPetukhov** [comment]
It occurred that RInside doesn't parse .Rprofile, so it's able to work only with packages, installed to the default path. Fixed it in 954ce6279e89478d79ac7f9222d8d010e46f92c8.
